### PR TITLE
[ZEPPELIN-5891] Method contains useless variables

### PR DESCRIPTION
**Pull request by NicolasGong**
### What is this PR for?
Method contains useless variables

### What type of PR is it?
Improvement

### What is the Jira issue?
* [ZEPPELIN-5891]

### CODE REVIEW

**Comment by NicolasGong** [comment]
> Very strange, we want to clone the metadata. How does the code work without a source NoteId?

I think this method is to put the new NoteId and AuthenticationInfo into the cache. This method does not require sourceNoteId.

**Comment by Reamer** [comment]
I think we can delete the `cloneNoteMeta` method completely and replace it with `createNoteAuth`.

In my opinion, the original rights should not be transferred from the old Note to the new Note. It makes sense to store the roles of the user who is currently cloning the note.

**Comment by NicolasGong** [comment]
I agree with you and resubmitted the code.

File: zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/AuthorizationService.java [hunk 1]
```diff
@@ -93,12 +93,7 @@ public void createNoteAuth(String noteId, AuthenticationInfo subject) {
     NoteAuth noteAuth = new NoteAuth(noteId, subject, conf);
     this.notesAuth.put(noteId, noteAuth);
   }
-
-  public void cloneNoteMeta(String noteId, String sourceNoteId, AuthenticationInfo subject) {
-    NoteAuth noteAuth = new NoteAuth(noteId, subject, conf);
-    this.notesAuth.put(noteId, noteAuth);
-  }
-
+
```
Comment (Reamer) [inline zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/AuthorizationService.java:96]: I suspect spaces here.

**Review by Reamer — CHANGES_REQUESTED** [review]
Please check whether the blanks are present.

File: zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/AuthorizationService.java (hunk 1 above)
Comment (NicolasGong) [reply zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/AuthorizationService.java]: I removed extra spaces.

**Review by Reamer — APPROVED** [review]
LGTM

**Comment by NicolasGong** [comment]
Some checks were not successful,but i don't know why. These errors do not appear to be related to this commit。

**Pull request merged**